samples/tab-mail/ts/src/index.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
```

Comment:
ESBuild implementation is also missing. Kindly refer Hari PR if you need.

Reply:
ES build is not needed for React samples I think, I came to know form jegadesh discussed with prithvi.